apps/consumer-client/src/pages/examples/add-pcd.tsx

```diff
@@ -637,7 +637,7 @@ async function addGPCPCD(
       argumentType: ArgumentTypeName.String,
       value: gpcConfig
     },
-    pod: {
+    [`${PODPCD_ARG_PREFIX}_pod0`]: {
```

Comment:
There needs to be some way of exercising multiple PODs from consumer-client to validate this, particularly the UI aspects.